packages/puppeteer-core/src/cdp/ChromeTargetManager.ts

```diff
@@ -358,10 +359,7 @@ export class ChromeTargetManager
     // `this.#connection.isAutoAttached(targetInfo.targetId)`. In the future, we
     // should determine if a target is auto-attached or not with the help of
     // CDP.
-    if (
-      targetInfo.type === 'service_worker' &&
-      this.#connection.isAutoAttached(targetInfo.targetId)
-    ) {
+    if (targetInfo.type === 'service_worker') {
```

Comment:
unrelated change: but this condition is redundant because of the check on line 351 (improved for readability)